unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/Wearables/WearablesFetchingHelper.cs

```diff
@@ -46,24 +47,26 @@ public static IEnumerator GetRandomCollections(int amount, bool ensureBaseWearab
             List<int> randomizedIndices = new List<int>();
             int randomIndex;
             bool addedBaseWearablesCollection = false;
+
             for (int i = 0; i < amount; i++)
             {
                 randomIndex = Random.Range(0, collections.Length);
+
                 while (randomizedIndices.Contains(randomIndex))
                 {
                     randomIndex = Random.Range(0, collections.Length);
                 }
 
-                if (ensureBaseWearables && collections[randomIndex].id == BASE_WEARABLES_COLLECTION_ID)
+                if (collections[randomIndex].id == BASE_WEARABLES_COLLECTION_ID)
```

Comment:
I had to change this logic because the existing one made no sense to me. Please ping me if you find anything wrong with this.

Reply:
@sandrade-dcl will have more insights in this